_im-plugin/ism/error-prevention/resolutions.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: default
+title: ISM error prevention resolutions
+parent: ISM error prevention
+nav_order: 5
+---
+
+# ISM error prevention resolutions
+
+Resolutions of errors for each validation rule action are listed below.
+
+---
+
+#### Table of contents
+1. TOC
+{:toc}
+
+
+---
+
+## The index is not the write index
+
+To confirm if one index is a write index, run the following request:
+
+```bash
+GET <index>/_alias?pretty
+```
+
+If the response does not contain `"is_write_index" : true`, the index is not a write index:
```

Comment:
This sentence defines what is not a write index but then provides an example for what is a write index.  For clarity, something like:
"If the response does not contain `"is_write_index" : true`, the index is not a write index. The following example confirms the index is a write index: